src/pluginList.ts

```diff
@@ -50,5 +50,9 @@ export default [
   {
     name : "web3-plugin-zksync",
     isFeatured : true
+  },
+    {
+    name : "@namespace-ens/web3-plugin-ens",
+    isFeatured : true
```

Comment:
```suggestion
    isFeatured : false
```
@branzinho great work for web3 plugin, before merging this PR could you make above change, as we set isFeatured true only for plugins that we have worked on and have reviewed \ publish.

Reply:
Sure! Updated the parameeter to false